app/mailers/previews/claim_mailer_preview.rb

```diff
@@ -0,0 +1,29 @@
+class ClaimMailerPreview < ActionMailer::Preview
+  def submitted
+    ClaimMailer.submitted(claim)
+  end
+
+  def approved
+    ClaimMailer.approved(claim)
+  end
+
+  def rejected
+    ClaimMailer.rejected(claim)
+  end
+
+  def payment_confirmation
+    payment = FactoryBot.build(:payment, :paid, claim: claim, created_at: DateTime.now)
```

Comment:
I hadn't considered using factories here in a mailer! Whilst I like the idea in principle, I think in practice it has some down sides. For example at the minute, every preview results in two new `LocalAuthority` records being created in the database, which isn't ideal. When creating previews, I normally use a scope to grab an appropriate existing record from the database and assume that one will exist. Perhaps instead of using factories, we can use the `seeds.rb` file to make sure the local database will aways contain appropriate records to preview the emails.

Reply:
Does `build` actually create database records though? I was under the impression that it just created them in memory.

Reply:
Yeah, in theory it's equivalent to `new` (vs `create`), but if you are building a record that has a `belongs_to` association on it, I think factory bot ends up saving the parent records.